auth-service/controllers/UserController.go

```diff
@@ -0,0 +1,17 @@
+package controllers
+
+import (
+	"github.com/Nguyenle23/Stocking-Microservices/services"
+	"github.com/gofiber/fiber/v2"
+)
+
+func UserController() *fiber.App {
+	app := fiber.New()
+
+	app.Get("/", services.GetAllUsers)
+	app.Get("/:id", services.GetUserByID)
+	app.Put("/:id", services.UpdateUser)
+	app.Delete("/:id", services.DeleteUserByID)
+
```

Comment:
follow the best practice in this link below
https://learn.microsoft.com/en-us/azure/architecture/best-practices/api-design